File: Client/v1/src/src/Requests/Users/CreateUserReq.js
class CreateUserReq {
    constructor(name, email, phone, userType, password) {
        this.name = name ? name : undefined;
        this.email = email ? email : undefined;
        this.phone = phone ? phone : undefined;
        this.userType = userType ? userType : undefined;
        this.password = password ? password : undefined;
    }
}

export default CreateUserReq;

File: Server/Tokenizer_V1/Tokenizer_V1/Requests/Users/CreateUserReq.cs
namespace Tokenizer_V1.Requests.Users
{
    public class CreateUserReq
    {
        public string Name { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string UserType { get; set; }
    }
}
